### ISSUE TITLE
Option not to render link on fail

### ISSUE THREAD

**Issue by YM-KA** [bradstiff/react-app-location#3]
When generating links by iterating data it would sometimes be preferred to be able tom omit links that do not match the schema. Now the whole page breaks with giving an error along the lines of "Warning: validateParams: id must match the following:"/-\w{19}/". See code excerpt below. 

```
...
//Definition
const someId = Yup.string().matches(/-\w{19}/)
export const MyLocation = new Location('/my/:id', {id : someId .required()})

...
//Link generation in component
props.things.map((my) => {
  return <li key={my.id} >
  ...
  { MyLocation.toLink( my.title, {id: my.id} ) }
  ...
```
If any single link fails validation the entire page breaks. 
The error stack shows: 
* Warning: validateParams: id must match the following: "/-\w{19}/"
* Warning: Failed prop type: The prop `to` is marked as required in `Link`, but its value is `null`.
* The above error occurred in the <Link> component:

Alternatively - would it be possible to define a default value that gets presented when validation fails? 

Kind regards
Kazuyuki

**Comment by bradstiff** [comment]
I could add an isValidParams method to the Location that you could use to conditionally render the link, or something else when invalid.

**Comment by YM-KA** [comment]
Thank you for the prompt reply! Yes, please - that would be fantastic! 👍

**Comment by bradstiff** [comment]
Implemented in v1.2.0, now on npm.

**Comment by YM-KA** [comment]
Many thanks, works like a charm!
